test/Driver/batch_mode_print_jobs.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: touch %t/file-01.swift %t/file-02.swift %t/file-03.swift
+// RUN: echo 'public func main() {}' >%t/main.swift
+//
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -driver-print-jobs | %FileCheck %s -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -###  | %FileCheck %s  -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -driver-print-jobs -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift 2>%t/shouldBeEmpty >/dev/null && test -z `cat %t/shouldBeEmpty`
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -### 2>%t/shouldBeEmpty > /dev/null && test -z `cat %t/shouldBeEmpty`
```

Comment:
Please put quotes around the sub-command, so that the argument doesn't get dropped entirely with a different shell.

Reply:
Done! Thanks.